src/lib.rs

```diff
@@ -64,6 +64,8 @@ pub struct Metadata {
     pub authors: Cow<'static, str>,
     /// The URL of the crate's website
     pub homepage: Cow<'static, str>,
+    /// The support information
+    pub supports: Cow<'static, str>,
```

Comment:
I feel like this should be `support` rather than `supports`, both in code and in the UI